### PR TITLE
{AzurePostgreSQL} include details about the underlying RESTAPI

### PR DESCRIPTION
**Pull request by navba-MSFT**
fixes Azure/azure-cli#27463

`az postgres flexible-server firewall-rule list ` command should mention about the rest api that is being called underneath.

**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AzurePostgreSQL

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py [hunk 1]
```diff
@@ -381,7 +381,7 @@
 
 helps['postgres flexible-server firewall-rule list'] = """
 type: command
-short-summary: List all firewall rules for a flexible server.
+short-summary: List all firewall rules for a flexible server. Run the command with `--debug` switch to identify the REST API that is invoked underneath and the response received from the service.
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:384]: `--debug` is a global parameter, not only for this command. There's no need to add a instruction here

Comment (navba-MSFT) [reply src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:384]: @evelyn-ys The OP of this thread https://github.com/Azure/azure-cli/issues/27463#issuecomment-1736889096  requested it to be added. Hence the PR was filed:

image

**Comment by evelyn-ys** [comment]
It's better to have a global doc for all global arguments. Not adding them to each command.

Thanks for great reminder!

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py (hunk 1 above)
Comment (evelyn-ys) [reply src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:384]: Please share https://learn.microsoft.com/en-us/cli/azure/get-started-with-azure-cli#globally-available-arguments with customer

**Pull request closed without merging**